src/Main.java

```diff
@@ -77,20 +89,17 @@ public static void main(String[] args) throws InterruptedException {
 				// Notice that in the real game the movements should be of 1/10
 				// cell. There is a method to do it.
 
+				
 				p1.move(lastAction, l1);
 				visualBoard.gb_moveSprite(0, p1.xPos, p1.yPos);
-
-				
 				p1.putBomb(lastAction, l1, timer);
 				
 				
-
-								
-
-//				if (lastAction.equals("space")) {
-//				visualBoard.gb_setSquareImage(p1.getxPos(), p1.getyPos(), "bomb1.gif");
-//			}
-				
+			}
+			
+			for (int ii=0; ii<balloon.length && timer%5 == 0; ii++) {

```

Comment:
we move the sprite in a random direction (first in the virtual board, then move it accordingly in the interface)